### DISCUSSION TITLE
Change in burn behavior due to mesh partitions

### DISCUSSION THREAD

**Discussion by sebastianmezaurotadze** [firemodels/fds#13238] [Q&A] [answered]
Id like to run simulations on very large domains (millions of cells) and use many partitions (~64) in order to speed up the compute time. I however notice that using a different number of subdomains (i.e. 64 vs 32 vs 16) can change burn behavior significantly. 

Any Insights into the following would be helpful:
- How information is transferred across mesh boundaries
- Why does using more subdomains change the simulation behavior
- How to mitigate this error from many domains 
- How to accelerate computation of large domains

Thanks

**Comment by rmcdermo** [comment]
Have a look at this chapter in the user guide: 21.1 Accuracy of the Pressure Solver

**Comment by mcgratta** [comment]
Can you post a plot of the two different heat release rate histories and the input files that generate them?

**Reply by sebastianmezaurotadze** [reply]
Found that lag in radiative heat transfer was likely the culprit. 

Here are some sample tests on smaller domains with HRR graphs comparing a single domain / split domain / split domain with higher radiation iterations to transfer radiative heat

![image_from_clipboard (1)](https://github.com/user-attachments/assets/cc607b8b-86dd-4c63-8e1f-4a57308a1470)

![image_from_clipboard](https://github.com/user-attachments/assets/06ad81c1-665e-4fa9-9b85-05886df374ca)

![image_from_clipboard (2)](https://github.com/user-attachments/assets/c36d5659-dc6f-498a-8a7a-3a4bca247546)


FDS Code:

[Single_1x_Domain_1x_Radiation_Iteration.txt](https://github.com/user-attachments/files/16407231/Single_1x_Domain_1x_Radiation_Iteration.txt)
[Split_4x_Domain_1x_Radiation_Iteration.txt](https://github.com/user-attachments/files/16407233/Split_4x_Domain_1x_Radiation_Iteration.txt)
[Split_4x_Domain_15x_Radiation_Iteration.txt](https://github.com/user-attachments/files/16407234/Split_4x_Domain_15x_Radiation_Iteration.txt)

**Reply by sebastianmezaurotadze** [reply]
Also, running a simple test with a flux gauge and a radiative particle separated by multiple mesh divisions on the  default FDS settings shows up to a 3 second delay even over short distance (2.8 m or 8 cells)

![image](https://github.com/user-attachments/assets/31df4b0d-5f94-4e46-a01e-e5b25d7814aa)

![image](https://github.com/user-attachments/assets/2ee3a7ad-062b-4c59-9a76-5e59f251e661)


![image](https://github.com/user-attachments/assets/38cc881f-039a-467f-b767-31e29dc79e91)


Any recommendations on how to optimize RADIATION_ITERATIONS as to maintain simulation accuracy without excessive runtime?

**Reply by mcgratta** [reply, marked as answer]
`RADIATION_ITERATIONS` is 1 be default. However, in cases of multiple meshes, you might want to increase it to 2 or 3 if it is very important that the radiation field be updated over the entire domain. Radiation intensity is exchanged at mesh boundaries once per solve. That means that a remote mesh does not "see" a heat source until FDS runs through X number of solves. Try setting it to 2 or 3 and see how that affects your problem.

**Comment by mcgratta** [comment]
Any progress on this?

**Reply by sebastianmezaurotadze** [reply]
Increasing radiation iterations helped, it didn't slow down my simulation significantly. This thread is already resolved.
